frontend/src/main-components/CreateCardContent.css

```diff
@@ -0,0 +1,110 @@
+html {
```

Comment:
Run a formatter.

Reply:
I'm using the prettier extension in VSCode, is there a setting I need to change?